plugins/inputs/mysql/v2/convert.go

```diff
@@ -70,12 +81,27 @@ func ParseValue(value sql.RawBytes) (interface{}, error) {
 }
 
 var GlobalStatusConversions = map[string]ConversionFunc{
-	"ssl_ctx_verify_depth": ParseInt,
-	"ssl_verify_depth":     ParseInt,
+	"innodb_available_undo_logs":    ParseUint,
+	"innodb_buffer_pool_pages_misc": ParseUint,
+	"innodb_data_pending_fsyncs":    ParseUint,
+	"ssl_ctx_verify_depth":          ParseUint,
+	"ssl_verify_depth":              ParseUint,
 }
 
 var GlobalVariableConversions = map[string]ConversionFunc{
-	"gtid_mode": ParseGTIDMode,
+	"delay_key_write":                  ParseString,        // ON, OFF, ALL
+	"enforce_gtid_consistency":         ParseString,        // ON, OFF, WARN
+	"event_scheduler":                  ParseBoolAsInteger, // YES, NO, DISABLED
+	"gtid_mode":                        ParseGTIDMode,
+	"have_openssl":                     ParseBoolAsInteger, // alias for have_ssl
+	"have_ssl":                         ParseBoolAsInteger, // YES, DISABLED
+	"have_symlink":                     ParseBoolAsInteger, // YES, NO, DISABLED
```

Comment:
event_scheduler and have_symlink have three states: YES, NO, DISABLED. If we use ParseBoolAsInteger, YES will return 1, but both NO and DISABLED return 0. This makes it impossible to know which of those two mysql returned. Is there a reason you didn't convert with ParseString in this case?

Similarly, have_ssl and have_openssl have YES and DISABLED. Using ParseBoolAsInteger isn't ambiguous here because there are only two options, but if we return 0 for NO everywhere else, maybe DISABLED should be some other value?

Reply:
Hey @reimda, I intended to make this PR as backwards-compatible as possible. `have_symlink` is probably almost always `YES` or `NO`, which used to be parsed as integer. `DISABLED` is only set if the MySQL server was started with a specific option. Changing this variable to string would probably cause a type conflict for most users, so I decided to keep the integer type and make `DISABLED` also 0.

Similar considerations apply to `have_ssl` as well.

However, I've got no strong opinion on this, so I could also parse all `DISABLED` fields as string.